content/java/concepts/array-list/terms/contains/contains.md

```diff
@@ -0,0 +1,57 @@
+---
+Title: '.contains()'
+Description: 'To check if an element is present in the list or not.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Arrays'
+  - 'Data Types'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The `.contains()` method is used for checking if the specified element is present in `ArrayList` or not. It will return true if the element is present; otherwise, it will return false.
```

Comment:
```suggestion
The **`.contains()`** method is used for checking if the specified element is present in `ArrayList` or not. It will return true if the element is present; otherwise, it will return false.
```
You should bold the first occurrence of the method name.